static-source/app/css/main.scss

```diff
@@ -0,0 +1,273 @@
+// Foundation
+@import 'util/util';
+@import 'foundation-settings';  // imported from my local dir
```

Comment:
What does that comment mean?

Reply:
that it imports `foundation-settings.scss` from the `static-source/app/css/` dir and not from a foundation-settings package or something... because we also have a foundation package in `node_modules`. I'll change this in the future when slowly moving away from foundation's code.